### ISSUE TITLE
can't regain the claimed  result 

### ISSUE THREAD

**Issue by jiaming-lee** [elliothe/CVPR_2019_PNI#1]
I run your code but can't regain the  claimed result  except the baseline model. Can you upload your pretrained model?

**Comment by elliothe** [comment]
@yangadan which result you cannot regain?

**Comment by jiaming-lee** [comment]
> @yangadan which result you cannot regain?

noise_resnet20 PNI-W model
**Test** Prec@1 83.090 Prec@5 99.080 Error@1 16.910
  **PGD Test** Prec@1 44.080 Prec@5 93.550 Error@1 55.920
  **FGSM Test** Prec@1 51.000 Prec@5 94.500 Error@1 49.000

**Comment by elliothe** [comment]
@yangadan Hi, I am sorry for the late reply. The pre-trained models for both layer-wise and channel-wise versions are uploaded at [Readme result section](https://github.com/elliothe/CVPR_2019_PNI#result).

I think there might be some setting problem. I just trained a new layerwise PNI model, and I can only get 84.11% for the clean accuracy. I will look at the training configuration and give you an update later.

**Comment by jiaming-lee** [comment]
> @yangadan Hi, I am sorry for the late reply. The pre-trained models for both layer-wise and channel-wise versions are uploaded at [Readme result section](https://github.com/elliothe/CVPR_2019_PNI#result).
> 
> I think there might be some setting problem. I just trained a new layerwise PNI model, and I can only get 84.11% for the clean accuracy. I will look at the training configuration and give you an update later.

Thank you
